hypertrace-graphql-span-processing-schema/src/main/java/org/hypertrace/graphql/spanprocessing/schema/rule/ExcludeSpanRuleInfo.java

```diff
@@ -16,21 +17,27 @@ public interface ExcludeSpanRuleInfo {
 
   @GraphQLField
   @GraphQLName(NAME_KEY)
+  @GraphQLDescription("Exclude span rule name")
   @GraphQLNonNull
   String name();
 
   @GraphQLField
   @GraphQLName(SPAN_FILTER_KEY)
+  @GraphQLDescription("Span processing rule filter contains filters based on span attributes")
   @GraphQLNonNull
   SpanProcessingRuleFilter spanFilter();
 
   @GraphQLField
   @GraphQLName(DISABLED_KEY)
+  @GraphQLDescription(
+      "Disabled field denotes whether the rule is disabled or not. By default it is enabled")
```

Comment:
The default isn't relevant since it's non null on read as this type describes. If there's a default on write, mention it there